src/components/Form/FormProvider.js

```diff
@@ -266,7 +266,14 @@ function FormProvider({validate, formID, shouldValidateOnBlur, shouldValidateOnC
                         // We delay the validation in order to prevent Checkbox loss of focus when
                         // the user is focusing a TextInput and proceeds to toggle a CheckBox in
                         // web and mobile web platforms.
+
+                        // Prevents React from resetting its properties
+                        event.persist();
```

Comment:
Crash on this line:

![crash](https://github.com/Expensify/App/assets/108292595/a71a8712-7b6b-4614-b292-b12ef17d89e3)

Reply:
Thanks @situchan, do you reproduction steps, or a link to an issue?

Reply:
Is there a bug opened yet?

Reply:
This isn't deployed to staging so not a deploy blocker yet. But will be on next deploy.
I believe no GH created because only QA team can report bugs caught on staging.

Reply:
Okay, thanks, I created an issue [here](https://github.com/Expensify/App/issues/31542).